client_cli/pytests/common/consts.py

```diff
@@ -15,11 +15,11 @@ class Stderr(Enum):
     Enum for standard error messages.
     """
 
-    CANNOT_BE_EMPTY = "cannot be empty\n\nFor more information, try '--help'.\n"
+    EMPTY = "mpty"
```

Comment:
typo?

Reply:
"mpty" was the intersection of various Empty error messages and was considered enough to identify the cause, but I revised the messages to contain "Empty"